src/store/modules/order/actions.ts

```diff
@@ -1111,10 +1166,60 @@ const actions: ActionTree<OrderState, RootState> = {
       logger.error('Something went wrong', err)
     }
 
+    current = await dispatch("fetchCurrentOrderCardActivationDetail", current);
+
     // updating the state with the updated orders information
     await dispatch('updateCurrent', current)
   },
 
+  async fetchCurrentOrderCardActivationDetail({ commit }, currentOrder) {
```

Comment:
Can't be use single function instead two separates functions fetchCurrentOrderCardActivationDetail and fetchOrdersCardActivationDetails?